README.md

```diff
@@ -94,13 +94,19 @@ There are several _optional_ configuration settings for adjusting this package t
 
 ### Directory
 
-The directory in the project to load image assets from. If you are using Exrm make sure you use a directory that is outputted to the projects `lib` directory after the release has been created.
+The directory in the project from which to load image assets.
+
+If you are using Exrm/Distillery, make sure you use a directory that is outputted to the projects `lib` directory after the release has been created.
 
 ```elixir
+# If you are using the standard way
+config :phoenix_inline_svg, dir: "/assets/somewhere/"
```

Comment:
Regarding your [comment](https://github.com/nikkomiu/phoenix_inline_svg/pull/20#issuecomment-506916951), shouldn't the example use a relative path?
```suggestion
config :phoenix_inline_svg, dir: "./assets/somewhere/"
```

Reply:
I think you're right, it should either be `"./assets/somewhere"` or `"assets/somewhere"`